src/dataUriTextDocumentContentProvider.ts

```diff
@@ -139,14 +139,18 @@ export class DataUriTextDocumentContentProvider implements TextDocumentContentPr
     }
 
     public async provideTextDocumentContent(uri: Uri): Promise<string> {
-        const filename = decodeURIComponent(uri.authority);
+        let fileName = decodeURIComponent(uri.authority);
         const query = querystring.parse<QueryDataUri>(uri.query);
         query.viewColumn = query.viewColumn || ViewColumn.Active.toString();
-        const document = vscode.workspace.textDocuments.find(e => e.fileName.toLowerCase() === filename.toLowerCase());
-        if (!document) {
-            return 'ERROR: Can no longer find document in editor: ' + filename;
+        let glTFContent: string;
+        const document = vscode.workspace.textDocuments.find(e => e.fileName.toLowerCase() === fileName.toLowerCase());
```

Comment:
If I recall correctly, that's why this is explicitly case-insensitive.  The `filename` comes from decoding `uri.authority`, which is lowercased.  Perhaps we need to move the filename out of `uri.authority` and over to `uri.path` which is case-sensitive.  Would need some new way to split the file path from the JSON path in that case.

Reply:
Right I see... we could use the fragment or query string for the path... I'll see what I can do.